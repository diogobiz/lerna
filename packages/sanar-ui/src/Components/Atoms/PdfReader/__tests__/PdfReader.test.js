
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESPdfReader from '../'

it('renders correctly', () => {
    const component = (
        <ESPdfReader>
            <h1>ESPdfReader</h1>
        </ESPdfReader>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
