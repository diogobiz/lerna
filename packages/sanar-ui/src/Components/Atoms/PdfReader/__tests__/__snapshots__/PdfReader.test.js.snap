// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-pdf-reader"
>
  <iframe
    onCompositionStart={true}
    onLoad={[Function]}
    src="https://drive.google.com/viewerng/viewer?url=undefined?pid=explorer&efh=false&a=v&chrome=false&embedded=true"
  />
  <div
    className="es-pdf-reader__footer"
  >
    <div
      className="es-typography es-typography--light es-typography--caption"
      data-testid="es-typography"
    >
      pdfReader.problemRenderingPdf
    </div>
    <button
      className="es-button es-button__xsmall es-button__variant--text secondary bold ant-btn"
      data-testid="es-button"
      onClick={[Function]}
      type="button"
    >
      <span>
        pdfReader.clickAndDownload
      </span>
    </button>
  </div>
</div>
`;
